### PR TITLE
protobuf 25.3

### PR DESCRIPTION
**Pull request by ik11235**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ZhongRuoyu** [comment]
There are a couple of merge conflicts with #162434 but we'll probably merge this first (if everything goes well) since `or-tools` doesn't build with the new `abseil`.

**Comment by p-linnane** [comment]
```
Error: 2 failed steps!
brew linkage --test ncnn
brew test --verbose ncnn
```
We can follow up on this after merging.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**
